src/navigation/routeKeys.ts

```diff
@@ -0,0 +1,8 @@
+export const dashboard = 'Dashboard' as const;
```

Comment:
what is `as const`?

Reply:
It makes the variable's type to be literal string. In this case `dashboard`'s type is `'Dashboard'` instead of `string`.

It's a shortcut for: `const dashboard: 'Dashboard' = 'Dashboard'`